server/rooms.ts

```diff
@@ -113,7 +113,6 @@ export interface RoomSettings {
 	showEnabled?: GroupSymbol | true;
 	permissions?: {[k: string]: GroupSymbol};
 	repeats?: RepeatedPhrase[];
-	quotes?: {userid: string, quote: string, date: number}[];
```

Comment:
This looks like an artifact from an unrelated PR?

Reply:
You're commenting on the wrong diff (it even says "Outdated"). That's in fact an artifact from an unrelated PR brought in by merging master, but it was merged correctly.

Reply:
Ah, my bad.